policy/modules/contrib/sandbox.te

```diff
@@ -49,7 +49,7 @@ corecmd_exec_all_executables(sandbox_domain)
 
 dev_dontaudit_getattr_all(sandbox_domain)
 
-files_rw_all_inherited_files(sandbox_domain, -exec_type -configfile -usr_t -lib_t -locale_t -var_t -var_run_t -device_t -rpm_log_t )
+files_rw_all_inherited_files(sandbox_domain, `-exec_type -configfile -usr_t -lib_t -locale_t -var_t -var_run_t -device_t -rpm_log_t')
 corecmd_entrypoint_all_executables(sandbox_domain)
```

Comment:
I think it might have been complaining about the trailing whitespace and that the `' is probably not needed (it might even cause unexpected results)

```
- files_rw_all_inherited_files(sandbox_domain, -exec_type -configfile -usr_t -lib_t -locale_t -var_t -var_run_t -device_t -rpm_log_t )
+ files_rw_all_inherited_files(sandbox_domain, -exec_type -configfile -usr_t -lib_t -locale_t -var_t -var_run_t -device_t -rpm_log_t)
```

Reply:
No, removing the trailing space doesn't silence the warning... But maybe the check is simply wrong here and it's OK without the quotes. IIUC, the only difference quotes make is that they suppresses macro expansion and special characters, plus allow having a newline character in the argument. I guess I'll just remove the extra space and suppress W-007 for this line...

Reply:
Yes, this interface call is a bit of a hack no matter how you look at it. There is no easy way to address this cleanly given the current limitations of m4.

I guess the thing is that the interface expects 2 param's and that the easiest way out here was to trick it to accept more than 2 param's. A type attribute could have been created and then be referenced as the second parameter but that would also present difficulties.

Reply:
The warning can be avoid by grouping the arguments:

```
files_rw_all_inherited_files(sandbox_domain, { -exec_type -configfile -usr_t -lib_t -locale_t -var_t -var_run_t -device_t -rpm_log_t })
```

Reply:
Indeed, that seems to generate the same rules. Although depending on the interpretation of the `{}` grouping one might expect `{ A { -B -C } }` to yield `A | (~B | ~C)`, it seems the parser just discards the `{}` and subtracts the union of the "negative" sets from the union of the "positive" sets (or whatever the logic is).

Thanks for the suggestion!